app/views/projects.html

```diff
@@ -8,83 +8,137 @@
+                                  </button>
+                                </div>
+                              </div>
+                            </form>
+                            <span class="vertical-divider"></span>
+                            <div class="projects-sort">
+                              <div pf-sort config="sortConfig" class="sort-controls"></div>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                    <div ng-if="(projects | findProject:search.text).length === 0" class="text-center">
+                      <h2>{{emptyMessage}}</h2>
+                    </div>
+                    <div class="list-group list-view-pf projects-list">
+                      <div ng-repeat="project in projects | findProject:search.text" class="list-group-item project-info tile-click">
+                        <div row class="list-view-pf-actions project-actions" ng-if="project.status.phase == 'Active'">
+                          <span class="fa-lg project-action-item">
+                            <a ng-href="project/{{project.metadata.name}}/edit" class="action-button">
+                              <i class="fa fa-pencil" aria-hidden="true"></i>
+                              <span class="sr-only">Edit Project</span>
+                            </a>
+                          </span>
+                          <span>
+                            <delete-link
+                              class="fa-lg project-action-item"
+                              kind="Project"
+                              resource-name="{{project.metadata.name}}"
+                              project-name="{{project.metadata.name}}"
+                              display-name="{{(project | displayName)}}"
+                              type-name-to-confirm="true"
+                              stay-on-current-page="true"
+                              alerts="alerts"
+                              button-only>
+                            </delete-link>
+                          </span>
+                        </div>
+                        <div class="list-view-pf-main-info">
+                          <div class="list-view-pf-description project-names">
+                            <div class="list-group-item-heading project-name-item">
+                              <h2 class="h1">
+                                <a class="tile-target" href="project/{{project.metadata.name}}">{{project.metadata.annotations["openshift.io/display-name"] || project.metadata.name}}</a> <!-- {{(project | uniqueDisplayName : projects)}} -->
+                                <span ng-if="project.status.phase != 'Active'"
+                                  data-toggle="tooltip"
+                                  title="This project has been marked for deletion."
+                                  class="pficon pficon-warning-triangle-o"></span>
```

Comment:
With the list view, is there a chance the icon can wrap on a line by itself below the project name?

I wonder if we still need this since we're watching projects, and it will be removed from the list pretty quickly. @jwforres opinion?